apps/dashboard/app/helpers/application_helper.rb

```diff
@@ -87,6 +87,15 @@ def icon_tag(icon_uri, classes: ['app-icon'])
     end
   end
 
+  def pathselector_favorites(favorites)
```

Comment:
I don't think you actually needed to move this. IIRC every helper is available to every page even if they're in separate files.